doc/schemas/listpeers.schema.json

```diff
@@ -165,954 +164,6 @@
                 }
               ]
             }
-          },
-          "channels": {
```

Comment:
Instead of removing everything from schema at once, would you please just mark as "deprecated"?

Reply:
it is deprecated in cln, but I do not know any type of way to deprecate it in the schema, maybe @rustyrussell know this?

Reply:
But I think this change will make some pain when you will go to generate some type? I think here the best solution is use a general call

Reply:
I've just want this change to follow a general field-deprecation cycle described in https://github.com/ElementsProject/lightning/blob/master/doc/STYLE.md#changing-json-apis
So maybe create another PR just marking this field as deprecated and merge that first?

If that causes too much pain in msggen... well, I suppose there is no reason to hurry in deprecating this field. So we should brush up msggen first?

Reply:
Note: I made a PR which changes msggen to allow more gradual deprecation cycle: https://github.com/ElementsProject/lightning/pull/5536

Reply:
Wait, I do not follow in this PR the deprecation rules? I think I follow

```
if (deprecated_apis) {
		json_array_start(response, "channels");
		json_add_peerchannels(ld, response, p);
		json_array_end(response);
	}
```

I do not know only how express this inside the json schema, do you have any idea?

Reply:
a possibility is add a deprecated prefix in the description of the json schema filed maybe? but I do not know, I do not think deeply to this idea

Reply:
> a possibility is add a deprecated prefix in the description of the json schema filed maybe?

Yes, that is what I was expecting. For 1. deprecation of rpc implementation, I think you are doing fine. but for 2. deprecation of rpc json schema, there are also guideline to annotate deprecated fields https://github.com/ElementsProject/lightning/blob/master/doc/schemas/WRITING_SCHEMAS.md#how-to-write-a-schema

(I should have posted this link at first, sorry.)

Reply:
You are right! thanks for this catch, I was assuming that we have no deprecated in the docs and we can not deprecate a schema!

Thanks!